corehq/apps/export/utils.py

```diff
@@ -13,6 +13,15 @@
 )
 
 
+def is_occurrence_deleted(last_occurrences, app_ids_and_versions):
```

Comment:
I debated it, but the function is tested since we have a few testes ensuring that columns get properly marked as deleted. Though might be good to isolate just this function